test/libsolidity/smtCheckerTests/types/event_with_rational_size_array.sol

```diff
@@ -0,0 +1,2 @@
+pragma experimental SMTChecker;
+contract a { event b(uint[(1 / 1)]); }
```

Comment:
@chriseth here, the type of the `BinaryExpression` `1 / 1` is `nullptr`.

Reply:
I can also change the SMTChecker to not go down after even definitions if that's preferred